src/core/server/graph/schema/schema.graphql

```diff
@@ -1269,6 +1269,38 @@ type StoryMessageBox {
   content: String
 }
 
+enum GIPHY_RATING {
+  G
+  PG
+  PG13
+  R
+}
+
+################################################################################
+## Embed Links
+################################################################################
+
+type EmbedConfiguration {
+  """
+  twitter when true will enable twitter link embeds in the stream.
+  """
+  twitter: Boolean!
+
+  """
+  youtube when true will enable youtube link embeds in the stream.
+  """
+  youtube: Boolean!
+
+  """
+  giphy
+  """
+  giphy: Boolean!
+
+  giphyMaxRating: GIPHY_RATING!
+
+  giphyAPIKey: String
```

Comment:
I would suggest breaking these out to `TwitterEmbedConfiguration`, `YouTubeEmbedConfiguration`, and `GiphyEmbedConfiguration` to standardize the `enabled: Boolean!` and fixing the `giphy*` prefixing.